python/pyspark/sql/connect/proto/base_pb2.pyi

```diff
@@ -2869,6 +2869,59 @@ class FetchErrorDetailsResponse(google.protobuf.message.Message):
             self, oneof_group: typing_extensions.Literal["_file_name", b"_file_name"]
         ) -> typing_extensions.Literal["file_name"] | None: ...
 
+    class QueryContext(google.protobuf.message.Message):
```

Comment:
qq; could we do this in Python client too?

Reply:
Not yet if I understand correctly. @gatorsmile has some concern on whether exposing QueryContext in the PySpark Exception APIs makes sense for non-sql PySpark exceptions. There is some ongoing discussion for this.